actionview/lib/action_view/helpers/translation_helper.rb

```diff
@@ -37,7 +37,8 @@ module TranslationHelper
       # you know what kind of output to expect when you call translate in a template.
       def translate(key, options = {})
         options = options.dup
-        options[:default] = wrap_translate_defaults(options[:default]) if options[:default]
+        remaining_defaults = Array(options.delete(:default))
+        options[:default] = remaining_defaults.shift if remaining_defaults.first.kind_of? String
```

Comment:
@jcoyne do you remember why you restricted `remaining_defaults` to Strings? The i18n gem allows you to pass Symbols and Procs as `:default` values too, but by omitting them from `remaining_defaults` here they get ignored.